templates/im_calc.sl.template

```diff
@@ -34,13 +32,16 @@ echo ___calculating simulations___
         fi
         #TODO: adjust this temporary code to use the update_db once merged
         cd {{mgmt_db_location}}/Runs/{{fault_name}}/{{sim_name}}
-        fd_name=`python -c "from qcore import utils; p = utils.load_yaml('sim_params.yaml'); print p['FD_STATLIST']"`
+        fd_name=`python -c "from qcore import utils; params = utils.load_sim_params('{{sim_dir}}/../sim_params.yaml'); print(params.FD_STATLIST)"`
```

Comment:
Is there a bash way to do this that is slightly more elegant than calling a python command like this? Even if we create our own scriptlet to return the value of a yaml file it would be nicer.